presto-delta/src/test/java/com/facebook/presto/delta/AbstractDeltaDistributedQueryTestBase.java

```diff
@@ -0,0 +1,137 @@
+    }
+
+    private static DistributedQueryRunner createDeltaQueryRunner(Map<String, String> extraProperties)
+            throws Exception
+    {
+        Session session = testSessionBuilder()
+                .setCatalog(DELTA_CATALOG)
+                .setTimeZoneKey(TimeZoneKey.UTC_KEY)
+                .build();
+
+        DistributedQueryRunner queryRunner = DistributedQueryRunner.builder(session)
+                .setExtraProperties(extraProperties)
+                .build();
+
+        // Install the TPCH plugin for test data (not in Delta format)
+        queryRunner.installPlugin(new TpchPlugin());
+        queryRunner.createCatalog("tpch", "tpch");
+
+        Path dataDir = queryRunner.getCoordinator().getBaseDataDir().resolve("delta_metadata");
+        Path catalogDir = dataDir.getParent().resolve("catalog");
+
+        // Install a Delta connector catalog
+        queryRunner.installPlugin(new DeltaPlugin());
+        Map<String, String> deltaProperties = ImmutableMap.<String, String>builder()
+                .put("hive.metastore", "file")
+                .put("hive.metastore.catalog.dir", catalogDir.toFile().toURI().toString())
+                .build();
+        queryRunner.createCatalog(DELTA_CATALOG, "delta", deltaProperties);
+
+        // Install a Hive connector catalog that uses the same metastore as Delta
+        // This catalog will be used to create tables in metastore as the Delta connector doesn't
+        // support creating tables yet.
+        queryRunner.installPlugin(new HivePlugin("hive"));
+        Map<String, String> hiveProperties = ImmutableMap.<String, String>builder()
+                .put("hive.metastore", "file")
+                .put("hive.metastore.catalog.dir", catalogDir.toFile().toURI().toString())
+                .put("hive.allow-drop-table", "true")
+                .put("hive.security", "legacy")
+                .build();
+        queryRunner.createCatalog(HIVE_CATALOG, "hive", hiveProperties);
+
+        return queryRunner;
+    }
+
+    /**
+     * Register the given <i>deltaTableName</i> as <i>hiveTableName</i> in HMS using the Hive storage catalog.
+     * Hive and Delta catalogs share the same HMS in this test. Hive is used to register the tables as Delta
+     * connector doesn't have the write support yet.
+     *
+     * @param deltaTableName Name of the delta table which is on the classpath.
+     * @param hiveTableName  Name of the Hive table that the Delta table is to be registered as in HMS
+     */
+    protected void registerDeltaTableInHMS(String deltaTableName, String hiveTableName)
+    {
+        getQueryRunner().execute("CREATE SCHEMA IF NOT EXISTS hive." + DELTA_SCHEMA);
+        getQueryRunner().execute(format(
+                "CREATE TABLE hive.\"%s\".\"%s\" (dummyColumn INT) WITH (external_location = '%s')",
+                DELTA_SCHEMA,
+                hiveTableName,
+                pathToGoldenDeltaTable(deltaTableName)));
+    }
```

Comment:
Does it support reading table has already been created in Hive using `io.delta.hive.DeltaStorageHandler`？

https://github.com/delta-io/connectors#create-a-hive-table